clients/cli/src/main/java/org/apache/gravitino/cli/Main.java

```diff
@@ -75,7 +75,8 @@ protected static String resolveCommand(CommandLine line) {
         return action;
       }
     } else if (args.length == 1) {
-      return CommandActions.DETAILS; /* Default to 'details' command. */
+      /* Default to 'details' command. */
+      return line.hasOption(GravitinoOptions.HELP) ? CommandActions.HELP : CommandActions.DETAILS;
```

Comment:
Isn't the HELP option generic?
I mean, it could be applied to any command, not matter how many args are provide.

Reply:
There are two type of help. This is the a general help showing the basic CLI options. The other is help on each entity type.